<?xml version="1.0"?>

<launch>

  <node pkg="joy" type="joy_node" name="joy" />

  <node pkg="ros_teleop" type="teleop_joy.py" name="teleop_joy">
    <remap from="cmd_vel" to="/joy_vel"/>
    <rosparam>
      speed: 0.2
      turn: 0.1
    </rosparam>
  </node>

</launch>
